src/data/markdown/docs/30 xk6-browser/30 xk6-browser.md

```diff
@@ -37,6 +39,58 @@ If you're more adventurous or want to get the latest changes of the xk6-browser
+| launch(browser_type, [options]) | The first argument is required and the only `browser_type` that is currently accepted is `chromium`. Below you will find a list of all possible `options`. The `options` param is optional. |
+
+### Options
+
+| Method            | Description                                                                                                                                                                         |
+|-------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| args[]            | Extra command line arguments to include when launching browser process. A list of chromium flags can be found [here](https://peter.sh/experiments/chromium-command-line-switches/). |
+| debug             | All CDP messages and internal fine grained logs will be logged if set to `true`. Default is `false`.                                                                                |
+| devtools          | Open up developer tools in the browser by default. Default is `false`.                                                                                                              |
+| env               | Environment variables to set before launching browser process.                                                                                                                      |
+| executablePath    | Override search for browser executable in favor of specified absolute path.                                                                                                         |
+| headless          | Show browser GUI or not. Defaults to `true`.                                                                                                                                        |
+| ignoreDefaultArgs | Ignore any of the default arguments included when launching browser process.                                                                                                        |
+| proxy             | Specify to set browser's proxy config.                                                                                                                                              |
+| slowMo            | Slow down input actions and navigation by specified time. By default this is off.                                                                                                   |
+| timeout           | Default timeout to use for various actions and navigation. Default timeout is 30 seconds.                                                                                           |
+
+### Example
+
+<CodeGroup labels={[]}>
+
+<!-- eslint-skip -->
+
+```javascript
+import launcher from 'k6/x/browser';
+
+export default function () {
+  const browser = launcher.launch('chromium', {
+    args: [],
```

Comment:
Since it's empty, we can omit it.

Reply:
I'll try to find a suitable arg to place in there